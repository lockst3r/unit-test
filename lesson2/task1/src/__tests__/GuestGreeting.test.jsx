import { shallow } from "enzyme";
import GuestGreeting from "../GuestGreeting";
import React from "react";

describe("GuestGreeting", () => {
  it("Should display GuestGreeting", () => {
    const wrappedComponent = shallow(<GuestGreeting />);
    expect(wrappedComponent).toMatchSnapshot();
  });
});