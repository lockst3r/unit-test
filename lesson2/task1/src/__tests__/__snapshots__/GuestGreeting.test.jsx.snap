// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GuestGreeting Should display GuestGreeting 1`] = `
<h1>
  Hi stranger âœŒ Please login ðŸš€
</h1>
`;
